### PR TITLE
Added Mura Dataset

### PR DESCRIPTION
**Pull request by SoumyadeepJana**

### CODE REVIEW

**Comment by SoumyadeepJana** [comment]
Dataset Request https://github.com/tensorflow/datasets/issues/377
@Conchylicultor  @cyfra @rsepassi  Kindly Review :)

**Comment by SinghKislay** [comment]
@SoumyadeepJana so I had the same issue when I first uploaded. Tensorflow uses 2 spaces instead of 4. They'll ask you to change. Check [this](https://www.tensorflow.org/community/contribute/code_style).

File: tensorflow_datasets/image/mura.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            
+            features=tfds.features.FeaturesDict({
+                "image":tfds.features.Image(shape=(None, None, 3)),
+                "label": tfds.features.ClassLabel(names=['positive', 'negative']),
+                "type":tfds.features.Text(),
+            }),
+
+            supervised_keys=("image", "label"),
+            
+            urls=[_URL],
```
Comment (us) [inline tensorflow_datasets/image/mura.py]: This url should be dataset webpage not download url.

**Comment by us** [comment]
@SinghKislay still continue?

**Comment by SinghKislay** [comment]
@us yes, sure.

**Comment by us** [comment]
Okay please generate fake examples for testing.

**Comment by SinghKislay** [comment]
@us so I just went to check the dataset structure and found [https://stanfordmlgroup.github.io/competitions/mura/](this). Can we even use the dataset, as it mentions one cannot share the download link and you will only get the download link through email. What should I do?

**Comment by cyfra** [comment]
@SinghKislay - good point. Let's continue the discussion on #824

**Pull request closed without merging**